app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,70 @@
+class FoodsController < ApplicationController
+  before_action :set_food, only: %i[ show edit update destroy ]
+
+  # GET /foods or /foods.json
+  def index
+    @foods = Food.all
+  end
+
+  # GET /foods/1 or /foods/1.json
+  def show
+  end
+
+  # GET /foods/new
+  def new
+    @food = Food.new
+  end
+
+  # GET /foods/1/edit
+  def edit
+  end
+
+  # POST /foods or /foods.json
+  def create
+    @food = Food.new(food_params)
+
+    respond_to do |format|
+      if @food.save
+        format.html { redirect_to food_url(@food), notice: "Food was successfully created." }
+        format.json { render :show, status: :created, location: @food }
+      else
+        format.html { render :new, status: :unprocessable_entity }
+        format.json { render json: @food.errors, status: :unprocessable_entity }
+      end
+    end
+  end
+
+  # PATCH/PUT /foods/1 or /foods/1.json
+  def update
+    respond_to do |format|
+      if @food.update(food_params)
+        format.html { redirect_to food_url(@food), notice: "Food was successfully updated." }
+        format.json { render :show, status: :ok, location: @food }
+      else
+        format.html { render :edit, status: :unprocessable_entity }
+        format.json { render json: @food.errors, status: :unprocessable_entity }
+      end
+    end
+  end
```

Comment:
hi @Graycemuthui! 👋  Good job implementing the CRUD methods, but the _**Update**_ method is not included as per project requirements [here](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/buisness_requirements.md) . You can safely remove these lines. (you can also change line 2 after that if you prefer).

Reply:
@Graycemuthui you are right, it is additional knowledge, but we have to follow the project`s requirements : 
**-A food list (with all CRUD implementation, except for 'update').**

Reply:
Ok I see that you resolved the issue in the next branch , so I am closing this issue. Great job! 👍